lib/ProductOpener/Ingredients.pm

```diff
@@ -3757,7 +3757,13 @@ sub preparse_ingredients_text($$) {
 
 	$log->debug("preparse_ingredients_text - before language specific preparsing", { text => $text }) if $log->is_debug();
 
-	if ($product_lc eq 'es') {
+	if ($product_lc eq 'de') {
+		# deletes comma in "Bienenwachs, weiß und gelb" since it is just one ingredient
+		$text =~ s/Bienenwachs, weiß und gelb/Bienenwachs weiß und gelb/ig;
+		# deletes brackets in "Bienenwachs, weiß und gelb" since it is just one ingredient
+		$text =~ s/Bienenwachs (weiß und gelb)/Bienenwachs weiß und gelb/ig;
```

Comment:
The parenthesis in regular expressions are used to group patterns. If you want to match parenthesis, they need to be escaped.

```suggestion
		$text =~ s/Bienenwachs \(weiß und gelb\)/Bienenwachs weiß und gelb/ig;
```